pom.xml

```diff
@@ -39,6 +39,10 @@
     <checkstyle.logViolationsToConsole>true</checkstyle.logViolationsToConsole>
     <jacoco.haltOnFailure>true</jacoco.haltOnFailure>
     <spotbugs.failOnViolation>true</spotbugs.failOnViolation>
+    <!-- This property has to be overridden by each module -->
+    <root-base-dir>${project.basedir}</root-base-dir>
```

Comment:
Can you not consider using `${maven.multiModuleProjectDirectory}` instead - you'd probably not have to override for every module then either. See https://stackoverflow.com/questions/3084629/finding-the-root-directory-of-a-multi-module-maven-reactor-project for some different approaches. `multiModuleProjectDirectory` seems to work OK as of Maven [3.3.9](https://issues.apache.org/jira/browse/MNG-5786).

Reply:
@manstis 
Already tried, of course, but it does not works.